src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -171,7 +171,7 @@ def load_arguments(self, _):
     with self.argument_context('appconfig kv list') as c:
         c.argument('key', help='If no key specified, return all keys by default. Support star sign as filters, for instance abc* means keys with abc as prefix.')
         c.argument('label', help="If no label specified, list all labels. Support star sign as filters, for instance abc* means labels with abc as prefix. Use '\\0' for null label.")
-        c.argument('resolve_keyvault', arg_type=get_three_state_flag(), help="Resolve the content of key vault reference.")
+        c.argument('resolve_keyvault', arg_type=get_three_state_flag(), help="Resolve the content of key vault reference. This argument should NOT be specified along with --fields instead use --query for customized query.")
```

Comment:
> --fields instead [](start = 170, length = 17)

Nit: This can be broken into two sentences. "... with --fields. Instead, use ..."